### DISCUSSION TITLE
videos/ts/ts-intro/

### DISCUSSION THREAD

**Discussion by giscus** [zxuqian/zxuqian.cn#53] [Announcements]
# videos/ts/ts-intro/

TypeScript 官方定义是 JavaScript的、带有类型的超集，并且能够编译为普通的 JavaScript。这里有三个重点，一个是编译，一个是带有类型，一个是超集。

https://zxuqian.cn/videos/ts/ts-intro/

**Comment by hunt-alone** [comment]
嗯嗯
